runscope/runscope.py

```diff
@@ -8,7 +8,7 @@
      
 runscope_service_url = "https://api.runscope.com/buckets"
 cache_expiry_min = 60
-cache_file = "test_meta_info.txt"
+cache_file = "/tmp/test_meta_info.txt"
```

Comment:
This will likely not work on Windows? Perhaps take a cache directory as an optional script argument and use that if provided, but default to `/tmp` (And if possible some other default for Windows?)